src/services/services.ts

```diff
@@ -3659,9 +3659,9 @@ namespace ts {
                         return containingNodeKind === SyntaxKind.Parameter;
 
                     case SyntaxKind.AsKeyword:
-                        containingNodeKind === SyntaxKind.ImportSpecifier ||
-                        containingNodeKind === SyntaxKind.ExportSpecifier ||
-                        containingNodeKind === SyntaxKind.NamespaceImport;
+                        return containingNodeKind === SyntaxKind.ImportSpecifier ||

```

Comment:
@DanielRosenwasser  Yeah I tried that rule and it wasn't usable with some of our patterns. And yeah services is not linted because I didn't turn on linting for any files I didn't clean up yet.